idl/github.com/uber/cadence/shared.thrift

```diff
@@ -1591,4 +1591,16 @@ struct ReapplyEventsRequest{
   10: optional string domainName
   20: optional WorkflowExecution workflowExecution
   30: optional DataBlob events
+}
+
+// SupportedClientVersions contains the support versions for client library
+struct SupportedClientVersions{
+  10: optional string gosdk
+  20: optional string javasdk
+  30: optional string cli
+}
+
+// DeploymentInfo contains information about cadence deployment
+struct DeploymentInfo{
+  10: optional SupportedClientVersions supportedClientVersions
```

Comment:
No cluster membership information?

Reply:
I am starting with just one at the beginning and add to it on next PR(s).